downsampling/generated_manifest.yml

```diff
@@ -0,0 +1,19 @@
+# This is a file generated by the scripts/generate script
+---
+  project-name: downsampling
+  templates:
+    - buckets.yml
```

Comment:
One thing I highly recommend  is to make these files an object like:

```yaml
templates:
   - template_file: buckets.yml
```

this allows you flexibility for the eventual changes that are coming. Perhaps the user can add a skip resource(s) or skip resource kind(s) to each template. Something along these lines is an extension. Whereas you may have a v2 of this that changes shape, and changing the shape of a v1 to a v2 is pretty simple given it is likely just a field conversion.

Reply:
awesome! i love comments like this.

Reply:
actually, I mispoke here. `- file: buckets.yml`, hte template prefix is redundant